custom/etc/zone.json.tmpl

```diff
@@ -6,7 +6,7 @@
   "cpu_shares": 64,
   "customer_metadata": {},
   "delegate_dataset": true,
-  "dns_domain": "jenkins.joyent.us",
+  "dns_domain": "jenkins.smartos.org",
```

Comment:
I can see this being used in more contexts than just jenkins, so maybe make this like `retro.smartos.org`? Or `smartos.example.com`? It's not really going to be a useful value anyway.

Reply:
Yeh good catch, fixed.

Reply:
I've also added a DNS entry for this so that the name effectively reserved.

```
$ dig +noall +question +answer -t any retro.smartos.org
;retro.smartos.org.             IN      ANY
retro.smartos.org.      3600    IN      TXT     "This record is used by smartos-retro HVM images"
retro.smartos.org.      3600    IN      A       127.0.0.1
```